coordinator/persistence-api/pom.xml

```diff
@@ -30,13 +30,6 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-      <!-- To upgrade StringTemplate version Duzzt uses, easiest to force here
```

Comment:
Since Duzzt itself removed, no need to use (or override version of ) StringTemplate library.